.eslintrc

```diff
@@ -151,7 +151,7 @@ rules:
   no-implicit-coercion: [1]
   no-implicit-globals: [0]
   no-implied-eval: [2]
-  no-inline-comments: [0]
+  no-inline-comments: [2]
```

Comment:
How about we change the rule and keep the changes I made so we don't have to do it again?